### ISSUE TITLE
Note and wiki page creation

### ISSUE THREAD

**Issue by Nepsi-cola** [shish/shimmie2#1182]
about:
  theme: "danbooru2"

versions:
  shimmie: "2.11.0-alpha"
  schema: 21
  php: "8.2.18"
  db: "mysql 10.11.6-MariaDB-cll-lve"
  os: "Linux 4.18.0-513.9.1.lve.el8.x86_64 #1 SMP Mon Dec 4 15:01:22 UTC 2023 x86_64"
  server: "LiteSpeed"

extensions:
  core: ["admin","alias_editor","bbcode","comment","download","et","ext_manager","four_oh_four","handle_pixel","help_pages","image","index","media","mime","post_lock","post_owner","post_source","post_tags","replace_file","setup","static_files","system","tag_list","upgrade","upload","user","user_config","view"]
  extra: ["approval","auto_tagger","autocomplete","blocks","bulk_actions","favorites","featured","forum","handle_mp3","handle_video","holiday","home","image_view_counter","notes","numeric_score","pm","pools","random_image","regen_thumb","relationships","report_image","site_description","sitemap","source_history","tag_categories","tag_history","tag_tools","wiki"]
  handled_mimes: ["image/jpeg","image/gif","image/png","image/webp","audio/mpeg","video/x-ms-asf","video/x-msvideo","video/x-flv","video/x-matroska","video/mp4","video/ogg","video/quicktime","video/webm"]

stats:
  images: 1442
  comments: 20
  users: 21

media:
  memory_limit: "8.0MB"
  disk_use: "1.4TB"
  disk_total: "2.0TB"

thumbnails:
  engine: "convert"
  quality: 75
  width: 192
  height: 192
  scaling: 100
  mime: "image/jpeg"

I'm currently running into two issues.

Navigating to and trying to edit wiki pages is broken. If I go to edit the wiki page for WORD, the page shoes up fine but when clicking the Edit button, fails with No handler could be found for the page 'wiki/WORD/edit'. BUT if I navigate to the page wiki/WORD/edit manually it allows me to edit the page like normal.
When attempting to create a new note I get the following error: SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data

I have applied commit 6cfbd64 before enabling nice URLs which fixed notes for me (thank you!), but now that I've gotten nice URLs enabled it no longer works and is giving me the same error as before. Everything else on the site works (as far as I can tell) using nice URLs but on the off chance it may be my htaccess implementation I've included it for scrutiny.

RewriteEngine On
RewriteBase /

# rather than link to images/ha/hash and have an ugly filename,
    # we link to images/hash/tags.ext; mod_rewrite splits things so
    # that shimmie sees hash and the user sees tags.ext
    RewriteRule ^_(images|thumbs)/([0-9a-f]{2})([0-9a-f]{30}).*$  data/$1/$2/$2$3 [L]
    # any requests for files which don't physically exist should be handled by index.php
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteRule ^(.*)$ index.php?q=$1&%{QUERY_STRING} "[L,QSA]"

# Rewrite the URL internally to index.php with q parameter
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteRule ^(.*)$ index.php?q=$1 [L,NE,B,QSA]

**Comment by Nepsi-cola** [comment]
After messing around with it a bit more, I've found that the POST to /note/create_note gets a 301 status (in picture). I don't know if that's relevant, but if I change the POST URL from website.com/note/create_note to website.com/index.php?q=note/create_note then the note is able to be created/deleted just fine. I've tried changing my htaccess rules to not redirect /note/create_note URLs as well as to only redirect that URL but nothing I've done has been able to fix it.

To reiterate, trying to save a note gives the error: SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
But changing the POST request to the old non-Nice URLs of website.com/index.php?q=note/create_note allows it to post just fine. I've already applied fix #1161.
